inc/ssr-handler.php

```
@@ -61,50 +60,34 @@ function djz_serve_ssr() {
-    $path = rtrim($path, '/');
-    if (empty($path)) {
-        $path = '/';
-    }
-
-    // Converter path para filename
-    $filename = djz_path_to_filename($path);
+    // Normalizar o path para encontrar o arquivo gerado pelo prerender.js
+    // O prerender.js cria pastas para cada rota e um index.html dentro.
+    // Ex: /about -> /dist/about/index.html
+    // Ex: /      -> /dist/index.html
+    $clean_path = trim($path, '/');
```

PHPMD (2.15.0)

[error] The variable $clean_path is not named in camelCase. (undefined)

(CamelCaseVariableName)